tests/inject/run.js

```diff
@@ -1,13 +1,20 @@
 // @ts-check
+const {getLogger, levels: {DEBUG, INFO}} = require('log4js');
 const karma = require('karma');
 const path = require('path');
 const {createEchoServer} = require('./echo-server');
 
 const ECHO_SERVER_PORT = 9966;
 
+process.env.NODE_OPTIONS = '--max_old_space_size=3072';
```

Comment:
Any particular need for this?

Reply:
Yes, when I do a test run, I get this:

```shell

_darkreader % npm run test:inject

> darkreader@4.9.45 test:inject
> node tests/inject/run.js

26 02 2022 23:21:16.834:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/customize.ts
26 02 2022 23:21:16.850:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/polyfills.ts
26 02 2022 23:21:21.688:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/utils/url.tests.ts
26 02 2022 23:21:30.096:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/image-analysis.tests.ts
If you do not supply "output.name", you may not be able to access the exports of an IIFE bundle.
26 02 2022 23:21:34.125:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/color.tests.ts
26 02 2022 23:21:34.126:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/inline-override.tests.ts
26 02 2022 23:21:34.126:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/link-override.tests.ts
26 02 2022 23:21:34.126:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/shadow-dom.tests.ts
26 02 2022 23:21:34.126:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/media-query.tests.ts
26 02 2022 23:21:34.127:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/fixes.tests.ts
26 02 2022 23:21:34.127:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/variables.tests.ts
26 02 2022 23:21:34.127:INFO [preprocessor.rollup]: Generating bundle for ./tests/inject/dynamic/style-override.tests.ts

<--- Last few GCs --->

[314843:0x570c7c0]    31477 ms: Mark-sweep (reduce) 2034.9 (2073.9) -> 2033.4 (2072.4) MB, 1055.6 / 0.0 ms  (+ 52.7 ms in 4 steps since start of marking, biggest step 13.7 ms, walltime since start of marking 1127 ms) (average mu = 0.198, current mu = 0.16[314843:0x570c7c0]    31493 ms: Scavenge (reduce) 2040.0 (2078.8) -> 2039.8 (2078.8) MB, 5.2 / 0.0 ms  (average mu = 0.198, current mu = 0.166) allocation failure 


<--- JS stacktrace --->

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
1: 0xb09980 node::Abort() [node]
2: 0xa1c235 node::FatalError(char const*, char const*) [node]
3: 0xcf77be v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
4: 0xcf7b37 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
5: 0xeaf3d5  [node]
6: 0xeafeb6  [node]
7: 0xebe3de  [node]
8: 0xebee20 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
9: 0xec1d9e v8::internal::Heap::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [node]
10: 0xe832da v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [node]
11: 0x11fc15c v8::internal::Runtime_AllocateInOldGeneration(int, unsigned long*, v8::internal::Isolate*) [node]
12: 0x15f0a99  [node]
^C|^\zsh: IOT instruction (core dumped)  npm run test:inject
npm run test:inject  79.03s user 4.23s system 139% cpu 59.519 total
```

Reply:
I meant to ask you if you had seen that before.

Reply:
Never seen it before... could you try and see if this error reproduces on master branch?

Reply:
Yeah, for me it's happening on master. I did run some profiling and didn't see anything weird. It just seems like the rollup process was the culprit.

Reply:
I'll grab another profiling report now..

Reply:
Usually I comment out the code coverage and the rollup, and just run the browser server, so I was new to the error

Reply:
There are things we could probably do with the typescript/rollup config to keep it under the 2mb. Or maybe start the echo server in a separate process?

Reply:
Here's the report:

<details>
<summary>Details</summary>

Statistical profiling result from isolate-0x68917f0-893758-v8.log, (32848 ticks, 42 unaccounted, 0 excluded).

 [Shared libraries]:
   ticks  total  nonlib   name
  25296   77.0%          /home/matt/.nvm/versions/node/v16.14.0/bin/node
      2    0.0%          [vdso]

 [JavaScript]:
   ticks  total  nonlib   name
    748    2.3%    9.9%  LazyCompile: *decode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:632:16
    463    1.4%    6.1%  LazyCompile: *encode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/sourcemap-codec/dist/sourcemap-codec.umd.js:78:17
    274    0.8%    3.6%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
    263    0.8%    3.5%  LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
    179    0.5%    2.4%  LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
    148    0.5%    2.0%  LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    134    0.4%    1.8%  LazyCompile: *addUneditedChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:299:65
    124    0.4%    1.6%  LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
    122    0.4%    1.6%  LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     81    0.2%    1.1%  LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     79    0.2%    1.0%  LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     79    0.2%    1.0%  LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:122:20
     78    0.2%    1.0%  LazyCompile: *traceMappings /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:13904:18
     65    0.2%    0.9%  LazyCompile: *append /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:61:9
     58    0.2%    0.8%  LazyCompile: *ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
     55    0.2%    0.7%  LazyCompile: *getObjectEntity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11077:20
     54    0.2%    0.7%  LazyCompile: *pp$9.eat /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:16823:20
     53    0.2%    0.7%  LazyCompile: *jsesc /home/matt/Projects/darkreader_ws/_darkreader/node_modules/jsesc/jsesc.js:76:15
     48    0.1%    0.6%  LazyCompile: *printJoin /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:332:12
     48    0.1%    0.6%  LazyCompile: *indent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1249:48
     41    0.1%    0.5%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:24:16
     40    0.1%    0.5%  LazyCompile: *bind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5066:9
     39    0.1%    0.5%  LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
     39    0.1%    0.5%  LazyCompile: *pp.readString /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21416:25
     35    0.1%    0.5%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:623:19
     32    0.1%    0.4%  LazyCompile: *setContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:135:20
     32    0.1%    0.4%  LazyCompile: *popContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:202:20
     32    0.1%    0.4%  LazyCompile: *NumericLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:191:24
     31    0.1%    0.4%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6544:12
     28    0.1%    0.4%  LazyCompile: *pp.readWord1 /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21618:24
     28    0.1%    0.4%  LazyCompile: *pp$5.parseMaybeAssign /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18566:33
     26    0.1%    0.3%  LazyCompile: *pp.readToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:20923:24
     25    0.1%    0.3%  LazyCompile: *pp.nextToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:20911:24
     25    0.1%    0.3%  LazyCompile: *buildPropertyMaps /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7312:22
     24    0.1%    0.3%  LazyCompile: *pp$5.parseObj /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19186:25
     24    0.1%    0.3%  LazyCompile: *ObjectEntity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7035:16
     23    0.1%    0.3%  LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
     22    0.1%    0.3%  LazyCompile: *pp$5.parseMaybeUnary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18687:32
     21    0.1%    0.3%  LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:154:20
     21    0.1%    0.3%  LazyCompile: *isFirstInContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:247:26
     21    0.1%    0.3%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/debug/src/common.js:125:9
     21    0.1%    0.3%  LazyCompile: *deoptimizeAllProperties /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7063:28
     20    0.1%    0.3%  LazyCompile: *token /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:94:8
     20    0.1%    0.3%  LazyCompile: *skipSpace /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2075:12
     20    0.1%    0.3%  LazyCompile: *pp$5.parseSubscript /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18782:31
     20    0.1%    0.3%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6520:15
     20    0.1%    0.3%  LazyCompile: *SourceMapGenerator_addMapping /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/node_modules/source-map/lib/source-map-generator.js:94:41
     19    0.1%    0.3%  LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
     19    0.1%    0.3%  LazyCompile: *create /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:38:9
     18    0.1%    0.2%  RegExp: [ !#-&\(-\[\]-_a-~]
     18    0.1%    0.2%  LazyCompile: *word /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:77:7
     18    0.1%    0.2%  LazyCompile: *shouldVisit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:24:14
     18    0.1%    0.2%  LazyCompile: *shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7923:21
     18    0.1%    0.2%  LazyCompile: *pp.getTokenFromCode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21169:31
     18    0.1%    0.2%  LazyCompile: *pp$5.parseIdent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19495:27
     17    0.1%    0.2%  LazyCompile: *pp.finishToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21017:26
     17    0.1%    0.2%  LazyCompile: *pp$5.parseExprAtom /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18848:30
     17    0.1%    0.2%  LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:226:20
     16    0.0%    0.2%  LazyCompile: *traceSegment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:13958:17
     16    0.0%    0.2%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11143:11
     16    0.0%    0.2%  LazyCompile: *pp.readNumber /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21363:25
     16    0.0%    0.2%  LazyCompile: *SourceMapGenerator_serializeMappings /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/node_modules/source-map/lib/source-map-generator.js:303:48
     15    0.0%    0.2%  LazyCompile: *readWord1 /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3099:12
     15    0.0%    0.2%  LazyCompile: *isIdentifierName /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/helper-validator-identifier/lib/identifier.js:58:26
     15    0.0%    0.2%  LazyCompile: *encode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:698:16
     15    0.0%    0.2%  LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
     15    0.0%    0.2%  LazyCompile: *_append /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:101:10
     15    0.0%    0.2%  LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
     13    0.0%    0.2%  LazyCompile: *scanJsDocToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11950:32
     13    0.0%    0.2%  LazyCompile: *pp$5.parseProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19203:30
     13    0.0%    0.2%  LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
     13    0.0%    0.2%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5088:15
     13    0.0%    0.2%  LazyCompile: *get node:internal/bootstrap/pre_execution:305:8
     13    0.0%    0.2%  LazyCompile: *deconflict /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6928:15
     12    0.0%    0.2%  LazyCompile: *scan /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11226:22
     12    0.0%    0.2%  LazyCompile: *nextToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2005:12
     12    0.0%    0.2%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     12    0.0%    0.2%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11131:15
     12    0.0%    0.2%  LazyCompile: *getTokenFromCode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2427:19
     11    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9312:12
     11    0.0%    0.1%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:539:19
     11    0.0%    0.1%  LazyCompile: *addUneditedChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1043:65
     10    0.0%    0.1%  LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7800:25
     10    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
      9    0.0%    0.1%  LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:627:36
      9    0.0%    0.1%  LazyCompile: *trackEntityAtPathAndGetIfTracked /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2097:37
      9    0.0%    0.1%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11069:11
      9    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11247:12
      9    0.0%    0.1%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6978:19
      9    0.0%    0.1%  LazyCompile: *crawl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:709:8
      9    0.0%    0.1%  LazyCompile: *bind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9180:9
      8    0.0%    0.1%  LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:95:21
      8    0.0%    0.1%  LazyCompile: *printList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:400:12
      8    0.0%    0.1%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9261:15
      8    0.0%    0.1%  LazyCompile: *getBinding /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:878:13
      8    0.0%    0.1%  LazyCompile: *exactSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:131:14
      8    0.0%    0.1%  LazyCompile: *bind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6468:9
      8    0.0%    0.1%  LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:45:26
      7    0.0%    0.1%  RegExp: ^(?:break|case|catch|continue|debugger|default|do|else|finally|for|function|if|return|switch|throw|try|var|while|with|null|true|false|instanceof|typeof|void|delete|new|in|this|const|class|extends|export|import|super)$
      7    0.0%    0.1%  LazyCompile: *shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5183:21
      7    0.0%    0.1%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6586:11
      7    0.0%    0.1%  LazyCompile: *parseIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12821:18
      7    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      7    0.0%    0.1%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:349:17
      7    0.0%    0.1%  LazyCompile: *deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6511:32
      7    0.0%    0.1%  LazyCompile: *computeLineOfPosition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10161:35
      7    0.0%    0.1%  LazyCompile: *cloneNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/clone/cloneNode.js:30:19
      7    0.0%    0.1%  LazyCompile: *StringLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:207:23
      7    0.0%    0.1%  LazyCompile: *Property /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11122:16
      6    0.0%    0.1%  RegExp: \b(preventAssignment|__DEBUG__|__WATCH__|__PORT__)\b(?!\.)(?!\s*=[^=])
      6    0.0%    0.1%  LazyCompile: *parseUpdate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11513:14
      6    0.0%    0.1%  LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8288:14
      6    0.0%    0.1%  LazyCompile: *parseBlockBody /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:13862:17
      6    0.0%    0.1%  LazyCompile: *match /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:1937:8
      6    0.0%    0.1%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/family.js:255:13
      6    0.0%    0.1%  LazyCompile: *finishToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2184:14
      6    0.0%    0.1%  LazyCompile: *createScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5085:16
      6    0.0%    0.1%  LazyCompile: *commaSeparator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:502:24
      6    0.0%    0.1%  LazyCompile: *_resyncList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:189:21
      6    0.0%    0.1%  LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
      6    0.0%    0.1%  LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6457:16
      6    0.0%    0.1%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/source-map.js:40:33
      5    0.0%    0.1%  LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      5    0.0%    0.1%  LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      5    0.0%    0.1%  LazyCompile: *isReferenced /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/isReferenced.js:8:22
      5    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9534:12
      5    0.0%    0.1%  LazyCompile: *getProgramParent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:802:19
      5    0.0%    0.1%  LazyCompile: *findVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6950:17
      5    0.0%    0.1%  LazyCompile: *findAncestor /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12537:26
      5    0.0%    0.1%  LazyCompile: *fields.shorthand.validate.Object.assign.type /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:640:98
      5    0.0%    0.1%  LazyCompile: *emitNodeList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:111151:30
      5    0.0%    0.1%  LazyCompile: *deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9473:32
      5    0.0%    0.1%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9210:19
      5    0.0%    0.1%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7092:19
      5    0.0%    0.1%  LazyCompile: *createToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:22338:29
      5    0.0%    0.1%  LazyCompile: *checkPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/lib/virtual-types.js:13:12
      5    0.0%    0.1%  LazyCompile: *bind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:45436:22
      5    0.0%    0.1%  LazyCompile: *Literal /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8252:1
      5    0.0%    0.1%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:66482:47
      5    0.0%    0.1%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36635:74
      4    0.0%    0.1%  LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:379:11
      4    0.0%    0.1%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8293:11
      4    0.0%    0.1%  LazyCompile: *pp$5.parsePropertyValue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19254:35
      4    0.0%    0.1%  LazyCompile: *parseMaybeUnary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11452:18
      4    0.0%    0.1%  LazyCompile: *newFn /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/visitors.js:210:26
      4    0.0%    0.1%  LazyCompile: *iterateCommentRanges /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10445:34
      4    0.0%    0.1%  LazyCompile: *isMatchingReference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:67527:37
      4    0.0%    0.1%  LazyCompile: *isFreshLiteralType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:61365:36
      4    0.0%    0.1%  LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      4    0.0%    0.1%  LazyCompile: *insertBefore /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/modification.js:25:22
      4    0.0%    0.1%  LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6554:25
      4    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      4    0.0%    0.1%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      4    0.0%    0.1%  LazyCompile: *getTypeOfSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:55361:33
      4    0.0%    0.1%  LazyCompile: *getPropertyOfType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57845:35
      4    0.0%    0.1%  LazyCompile: *getObjectEntity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8685:20
      4    0.0%    0.1%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/index.js:8:17
      4    0.0%    0.1%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/index.js:73:13
      4    0.0%    0.1%  LazyCompile: *disallowNamespaceReassignment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9373:34
      4    0.0%    0.1%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9463:19
      4    0.0%    0.1%  LazyCompile: *debug /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/index.js:158:8
      4    0.0%    0.1%  LazyCompile: *checkExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:77780:33
      4    0.0%    0.1%  LazyCompile: *_resyncKey /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:168:20
      4    0.0%    0.1%  LazyCompile: *_containerInsert /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/modification.js:51:26
      4    0.0%    0.1%  LazyCompile: *Scope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:275:14
      4    0.0%    0.1%  LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:30:20
      4    0.0%    0.1%  LazyCompile: *ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      4    0.0%    0.1%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:667:21
      4    0.0%    0.1%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:648:20
      4    0.0%    0.1%  Function: ^scanJsDocToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11950:32
      4    0.0%    0.1%  Function: ^indent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1249:48
      3    0.0%    0.0%  LazyCompile: *updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:7504:16
      3    0.0%    0.0%  LazyCompile: *token /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31140:23
      3    0.0%    0.0%  LazyCompile: *skipTrivia /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10275:24
      3    0.0%    0.0%  LazyCompile: *resolveNameHelper /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48310:35
      3    0.0%    0.0%  LazyCompile: *resolve node:path:1091:10
      3    0.0%    0.0%  LazyCompile: *pp$8.parseStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17016:31
      3    0.0%    0.0%  LazyCompile: *pp$5.parseExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18551:32
      3    0.0%    0.0%  LazyCompile: *pp$5.parseExprList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19443:30
      3    0.0%    0.0%  LazyCompile: *pp$2.startNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19702:26
      3    0.0%    0.0%  LazyCompile: *pipelineEmitWithComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112015:42
      3    0.0%    0.0%  LazyCompile: *pipelineEmit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108404:30
      3    0.0%    0.0%  LazyCompile: *parseSubscripts /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11550:18
      3    0.0%    0.0%  LazyCompile: *parseMaybeAssign /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:6024:19
      3    0.0%    0.0%  LazyCompile: *parseMaybeAssign /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11217:19
      3    0.0%    0.0%  LazyCompile: *next /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:1914:7
      3    0.0%    0.0%  LazyCompile: *join node:path:1166:7
      3    0.0%    0.0%  LazyCompile: *is_reference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5415:23
      3    0.0%    0.0%  LazyCompile: *isStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:4053:21
      3    0.0%    0.0%  LazyCompile: *isSimpleTypeRelatedTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:62874:39
      3    0.0%    0.0%  LazyCompile: *isDenylisted /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:70:22
      3    0.0%    0.0%  LazyCompile: *isCatchClause /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:438:23
      3    0.0%    0.0%  LazyCompile: *isBlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:393:26
      3    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      3    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      3    0.0%    0.0%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11809:15
      3    0.0%    0.0%  LazyCompile: *getVariableRespectingTDZ /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6619:29
      3    0.0%    0.0%  LazyCompile: *getTransformFlagsSubtreeExclusions /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:26563:48
      3    0.0%    0.0%  LazyCompile: *getSourceFileOfNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:14322:33
      3    0.0%    0.0%  LazyCompile: *getNodeLinks /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48016:30
      3    0.0%    0.0%  LazyCompile: *getFlowTypeOfReference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:68490:40
      3    0.0%    0.0%  LazyCompile: *getEmitFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:14594:26
      3    0.0%    0.0%  LazyCompile: *findApplicableIndexInfo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57889:41
      3    0.0%    0.0%  LazyCompile: *fields.Object.assign.name.validate.Object.assign.type /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:363:97
      3    0.0%    0.0%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8664:19
      3    0.0%    0.0%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11048:19
      3    0.0%    0.0%  LazyCompile: *createUnionOrIntersectionProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57603:51
      3    0.0%    0.0%  LazyCompile: *createHasEffectsContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:4593:33
      3    0.0%    0.0%  LazyCompile: *checkPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/lib/virtual-types.js:113:12
      3    0.0%    0.0%  LazyCompile: *bindEach /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43912:26
      3    0.0%    0.0%  LazyCompile: *applyDeoptimizations /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9354:25
      3    0.0%    0.0%  LazyCompile: *VariableDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:264:29
      3    0.0%    0.0%  LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11756:16
      3    0.0%    0.0%  LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11041:16
      3    0.0%    0.0%  LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:173:24
      3    0.0%    0.0%  Function: ^parseMaybeUnary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11452:18
      3    0.0%    0.0%  Function: ^getTypeOfSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:55361:33
      3    0.0%    0.0%  Function: ^emitNodeList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:111151:30
      3    0.0%    0.0%  Function: ^deoptimizeAllProperties /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7063:28
      3    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36635:74
      2    0.0%    0.0%  RegExp: \.0+$
      2    0.0%    0.0%  LazyCompile: *wrapped /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-node-resolve/dist/cjs/index.js:53:19
      2    0.0%    0.0%  LazyCompile: *visitNodes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29873:24
      2    0.0%    0.0%  LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:523:36
      2    0.0%    0.0%  LazyCompile: *unescapeLeadingUnderscores /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12584:40
      2    0.0%    0.0%  LazyCompile: *types$1.parenR.updateContext.types$1.braceR.updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18406:71
      2    0.0%    0.0%  LazyCompile: *types$1.parenL.updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18428:40
      2    0.0%    0.0%  LazyCompile: *tryGetTypeAtPosition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:75516:38
      2    0.0%    0.0%  LazyCompile: *toReferencedList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:5667:19
      2    0.0%    0.0%  LazyCompile: *step /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:45:18
      2    0.0%    0.0%  LazyCompile: *source /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:180:9
      2    0.0%    0.0%  LazyCompile: *some /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:945:18
      2    0.0%    0.0%  LazyCompile: *setScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:119:18
      2    0.0%    0.0%  LazyCompile: *scanIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11685:32
      2    0.0%    0.0%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7918:11
      2    0.0%    0.0%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11257:11
      2    0.0%    0.0%  LazyCompile: *queue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:75:8
      2    0.0%    0.0%  LazyCompile: *propagateChildrenFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:26548:36
      2    0.0%    0.0%  LazyCompile: *pp$9.checkExpressionErrors /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:16915:38
      2    0.0%    0.0%  LazyCompile: *pp$8.parseVarStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17300:34
      2    0.0%    0.0%  LazyCompile: *pp$8.parseVarId /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17452:27
      2    0.0%    0.0%  LazyCompile: *pp$8.parseBlock /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17363:27
      2    0.0%    0.0%  LazyCompile: *pp$7.checkLValSimple /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18239:32
      2    0.0%    0.0%  LazyCompile: *pp$5.parseParenAndDistinguishExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19029:51
      2    0.0%    0.0%  LazyCompile: *pp$2.startNodeAt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19706:28
      2    0.0%    0.0%  LazyCompile: *pp$2.copyNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19732:25
      2    0.0%    0.0%  LazyCompile: *parseTagComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36879:42
      2    0.0%    0.0%  LazyCompile: *parseSubscript /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11565:17
      2    0.0%    0.0%  LazyCompile: *parseOptional /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31403:31
      2    0.0%    0.0%  LazyCompile: *parseNonArrayType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:33054:35
      2    0.0%    0.0%  LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10154:14
      2    0.0%    0.0%  LazyCompile: *parseJSDocCommentWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36615:45
      2    0.0%    0.0%  LazyCompile: *parseExprOpRightExpr /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11389:23
      2    0.0%    0.0%  LazyCompile: *parseExprAtom /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11783:16
      2    0.0%    0.0%  LazyCompile: *parseExpected /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31256:31
      2    0.0%    0.0%  LazyCompile: *parseDelimitedList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:32196:36
      2    0.0%    0.0%  LazyCompile: *normalizeString node:path:66:25
      2    0.0%    0.0%  LazyCompile: *newBranch /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/source-coverage.js:69:12
      2    0.0%    0.0%  LazyCompile: *isType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/isType.js:10:16
      2    0.0%    0.0%  LazyCompile: *isNodeType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/introspection.js:58:20
      2    0.0%    0.0%  LazyCompile: *isExpressionNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:15897:30
      2    0.0%    0.0%  LazyCompile: *isClassExtendsClause /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:61:30
      2    0.0%    0.0%  LazyCompile: *initialise /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9332:15
      2    0.0%    0.0%  LazyCompile: *initialise /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8285:15
      2    0.0%    0.0%  LazyCompile: *includeVariableInModule /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:13078:28
      2    0.0%    0.0%  LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9324:25
      2    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8916:12
      2    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11200:12
      2    0.0%    0.0%  LazyCompile: *identity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:1723:22
      2    0.0%    0.0%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8063:15
      2    0.0%    0.0%  LazyCompile: *getSymbolLinks /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48010:32
      2    0.0%    0.0%  LazyCompile: *getSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48023:27
      2    0.0%    0.0%  LazyCompile: *getResolvedSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:67482:35
      2    0.0%    0.0%  LazyCompile: *getObjectFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:19382:28
      2    0.0%    0.0%  LazyCompile: *getDeclarationNodeFlagsFromSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72372:51
      2    0.0%    0.0%  LazyCompile: *getControlFlowContainer /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69569:41
      2    0.0%    0.0%  LazyCompile: *getCheckFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:19206:27
      2    0.0%    0.0%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:343:17
      2    0.0%    0.0%  LazyCompile: *forEachLeadingCommentToEmit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112276:45
      2    0.0%    0.0%  LazyCompile: *forEachChild /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29907:26
      2    0.0%    0.0%  LazyCompile: *exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:323:29
      2    0.0%    0.0%  LazyCompile: *emitPos /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112398:25
      2    0.0%    0.0%  LazyCompile: *eat /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:1928:6
      2    0.0%    0.0%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6505:19
      2    0.0%    0.0%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2150:19
      2    0.0%    0.0%  LazyCompile: *declareSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43614:31
      2    0.0%    0.0%  LazyCompile: *declare /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6474:12
      2    0.0%    0.0%  LazyCompile: *createIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31508:34
      2    0.0%    0.0%  LazyCompile: *createBaseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:22077:32
      2    0.0%    0.0%  LazyCompile: *coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      2    0.0%    0.0%  LazyCompile: *contains /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6925:13
      2    0.0%    0.0%  LazyCompile: *compareValues /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:1874:27
      2    0.0%    0.0%  LazyCompile: *bind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9426:9
      2    0.0%    0.0%  LazyCompile: *addDeclarationToSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43512:40
      2    0.0%    0.0%  LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:73:26
      2    0.0%    0.0%  LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:111:28
      2    0.0%    0.0%  LazyCompile: *Module._nodeModulePaths node:internal/modules/cjs/loader:629:37
      2    0.0%    0.0%  LazyCompile: *LocalVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6310:16
      2    0.0%    0.0%  LazyCompile: *FunctionScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7870:16
      2    0.0%    0.0%  LazyCompile: *BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7931:16
      2    0.0%    0.0%  Function: ^visitEachChild /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:87195:28
      2    0.0%    0.0%  Function: ^unescapeLeadingUnderscores /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12584:40
      2    0.0%    0.0%  Function: ^trampoline /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29813:28
      2    0.0%    0.0%  Function: ^scan /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11226:22
      2    0.0%    0.0%  Function: ^resolveNameHelper /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48310:35
      2    0.0%    0.0%  Function: ^resolveCall /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:74092:29
      2    0.0%    0.0%  Function: ^reportErrorResults /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:63442:44
      2    0.0%    0.0%  Function: ^render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9555:11
      2    0.0%    0.0%  Function: ^printAndIndentOnComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:359:27
      2    0.0%    0.0%  Function: ^pp$7.checkLValSimple /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18239:32
      2    0.0%    0.0%  Function: ^pp$5.parsePropertyValue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19254:35
      2    0.0%    0.0%  Function: ^pp$5.parseExprAtom /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18848:30
      2    0.0%    0.0%  Function: ^pipelineEmitWithHintWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108473:44
      2    0.0%    0.0%  Function: ^parseStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:35387:32
      2    0.0%    0.0%  Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      2    0.0%    0.0%  Function: ^parseExprAtom /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11783:16
      2    0.0%    0.0%  Function: ^newBranch /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/source-coverage.js:69:12
      2    0.0%    0.0%  Function: ^hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9505:15
      2    0.0%    0.0%  Function: ^getSignatureApplicabilityError /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73767:48
      2    0.0%    0.0%  Function: ^forEachChild /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29907:26
      2    0.0%    0.0%  Function: ^declareSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43614:31
      2    0.0%    0.0%  Function: ^createNodeArray /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:22042:33
      2    0.0%    0.0%  Function: ^createCallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:23388:38
      2    0.0%    0.0%  Function: ^consolidateInitializers /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6327:28
      2    0.0%    0.0%  Function: ^checkPropertyAccessibilityAtLocation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72415:54
      2    0.0%    0.0%  Function: ^checkIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69699:33
      2    0.0%    0.0%  Function: ^checkExpressionWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:77822:39
      2    0.0%    0.0%  Function: ^bindParameter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:46284:31
      2    0.0%    0.0%  Function: ^addParameterVariables /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7791:26
      2    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48958:112
      2    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43910:38
      2    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11180:28
      1    0.0%    0.0%  RegExp: e
      1    0.0%    0.0%  RegExp: ^in(stanceof)?$
      1    0.0%    0.0%  RegExp: ^0[box]
      1    0.0%    0.0%  RegExp: ^(?:enum|await|implements|interface|let|package|private|protected|public|static|yield)$ {1}
      1    0.0%    0.0%  RegExp: ^(?:enum|await|implements|interface|let|package|private|protected|public|static|yield)$
      1    0.0%    0.0%  RegExp: ^(?:break|case|catch|continue|debugger|default|do|else|finally|for|function|if|return|switch|throw|try|var|while|with|null|true|false|instanceof|typeof|void|delete|new|in|this|const|class|extends|export|import|super)$ {1}
      1    0.0%    0.0%  RegExp: ^(?:\/home\/matt\/Projects\/darkreader_ws\/_darkreader\/(?!\.{1,2}(?:\/|$))(?=.)[^/]*?\.ts(?:|x)+)$
      1    0.0%    0.0%  RegExp: ^(?:\.\.?(?:\/|$)|\/|([A-Za-z]:)?[/\\])
      1    0.0%    0.0%  RegExp: ^ +
      1    0.0%    0.0%  LazyCompile: *writeText /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:17919:27
      1    0.0%    0.0%  LazyCompile: *write /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:17929:23
      1    0.0%    0.0%  LazyCompile: *wrap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:173:5
      1    0.0%    0.0%  LazyCompile: *withTrackedEntityAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2104:28
      1    0.0%    0.0%  LazyCompile: *visitNodes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:87020:24
      1    0.0%    0.0%  LazyCompile: *visitNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29870:23
      1    0.0%    0.0%  LazyCompile: *visitEachChild /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:87195:28
      1    0.0%    0.0%  LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:138:20
      1    0.0%    0.0%  LazyCompile: *types$1.name.updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18468:38
      1    0.0%    0.0%  LazyCompile: *types$1.braceL.updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18418:40
      1    0.0%    0.0%  LazyCompile: *tryParseSemicolon /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31450:35
      1    0.0%    0.0%  LazyCompile: *traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:15383:26
      1    0.0%    0.0%  LazyCompile: *toReferencedListDeep /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10913:23
      1    0.0%    0.0%  LazyCompile: *startNodeAt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3829:14
      1    0.0%    0.0%  LazyCompile: *setParent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:20775:23
      1    0.0%    0.0%  LazyCompile: *semicolon /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:55:12
      1    0.0%    0.0%  LazyCompile: *scanString /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10861:28
      1    0.0%    0.0%  LazyCompile: *replaceWith /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/replacement.js:102:21
      1    0.0%    0.0%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9335:11
      1    0.0%    0.0%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7975:11
      1    0.0%    0.0%  LazyCompile: *render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5167:11
      1    0.0%    0.0%  LazyCompile: *registerBinding /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:557:18
      1    0.0%    0.0%  LazyCompile: *readTmplToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2920:16
      1    0.0%    0.0%  LazyCompile: *readString /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2889:13
      1    0.0%    0.0%  LazyCompile: *readInt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2678:10
      1    0.0%    0.0%  LazyCompile: *rangeEndIsOnSameLineAsRangeStart /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:19130:46
      1    0.0%    0.0%  LazyCompile: *propagateChildFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:26542:33
      1    0.0%    0.0%  LazyCompile: *processComment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:147:17
      1    0.0%    0.0%  LazyCompile: *printInnerComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:384:21
      1    0.0%    0.0%  LazyCompile: *pp.readToken_plus_min /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21091:33
      1    0.0%    0.0%  LazyCompile: *pp.readToken_eq_excl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21126:32
      1    0.0%    0.0%  LazyCompile: *pp.readInt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21295:22
      1    0.0%    0.0%  LazyCompile: *pp.next /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:20871:19
      1    0.0%    0.0%  LazyCompile: *pp$8.parseReturnStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17205:37
      1    0.0%    0.0%  LazyCompile: *pp$8.parseIfStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17196:33
      1    0.0%    0.0%  LazyCompile: *pp$5.parseLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19013:29
      1    0.0%    0.0%  LazyCompile: *pp$3.enterScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19609:27
      1    0.0%    0.0%  LazyCompile: *pp$3.declareName /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19624:28
      1    0.0%    0.0%  LazyCompile: *pp$2.finishNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19722:27
      1    0.0%    0.0%  LazyCompile: *pipelineEmitWithSourceMaps /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112340:44
      1    0.0%    0.0%  LazyCompile: *pipelineEmitWithHintWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108473:44
      1    0.0%    0.0%  LazyCompile: *parseTypeAnnotation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:33390:37
      1    0.0%    0.0%  LazyCompile: *parseTokenNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31430:32
      1    0.0%    0.0%  LazyCompile: *parseStatementContent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:13288:24
      1    0.0%    0.0%  LazyCompile: *parsePropertyDefinition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12470:26
      1    0.0%    0.0%  LazyCompile: *parseOptionalToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31410:36
      1    0.0%    0.0%  LazyCompile: *parseObjectLike /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12411:18
      1    0.0%    0.0%  LazyCompile: *parseList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31880:27
      1    0.0%    0.0%  LazyCompile: *parseIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31539:33
      1    0.0%    0.0%  LazyCompile: *parseFunctionBodyAndFinish /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12711:29
      1    0.0%    0.0%  LazyCompile: *parseFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:13980:16
      1    0.0%    0.0%  LazyCompile: *parseExprListItem /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12791:20
      1    0.0%    0.0%  LazyCompile: *parseCoverCallAndAsyncArrowHead /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11635:34
      1    0.0%    0.0%  LazyCompile: *parseBinaryExpressionOrHigher /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:33891:47
      1    0.0%    0.0%  LazyCompile: *onex /home/matt/Projects/darkreader_ws/_darkreader/node_modules/resolve/lib/async.js:199:26
      1    0.0%    0.0%  LazyCompile: *maybeTypeOfKind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:76485:33
      1    0.0%    0.0%  LazyCompile: *markDeclarationReached /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6583:27
      1    0.0%    0.0%  LazyCompile: *lookAhead /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31221:27
      1    0.0%    0.0%  LazyCompile: *isUnaryExpressionKind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13619:35
      1    0.0%    0.0%  LazyCompile: *isTupleType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:65806:29
      1    0.0%    0.0%  LazyCompile: *isSimpleParamList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12749:20
      1    0.0%    0.0%  LazyCompile: *isSelfReferenceLocation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48759:41
      1    0.0%    0.0%  LazyCompile: *isRelatedTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:63297:33
      1    0.0%    0.0%  LazyCompile: *isReadonlySymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:76250:34
      1    0.0%    0.0%  LazyCompile: *isReachableFlowNodeWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:68365:43
      1    0.0%    0.0%  LazyCompile: *isPossibleTDZ /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6557:18
      1    0.0%    0.0%  LazyCompile: *isOptionalCallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:1503:34
      1    0.0%    0.0%  LazyCompile: *isNewExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:768:25
      1    0.0%    0.0%  LazyCompile: *isListTerminator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31797:34
      1    0.0%    0.0%  LazyCompile: *isLineTerminator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3524:19
      1    0.0%    0.0%  LazyCompile: *isLineBreak /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10216:25
      1    0.0%    0.0%  LazyCompile: *isJSDocPropertyLikeTag /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13145:36
      1    0.0%    0.0%  LazyCompile: *isIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:618:22
      1    0.0%    0.0%  LazyCompile: *isFunctionLike /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13317:28
      1    0.0%    0.0%  LazyCompile: *isFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:4188:20
      1    0.0%    0.0%  LazyCompile: *isFor /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:4158:15
      1    0.0%    0.0%  LazyCompile: *isExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13651:26
      1    0.0%    0.0%  LazyCompile: *isEmitNotificationEnabled /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:107096:43
      1    0.0%    0.0%  LazyCompile: *isCallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:423:26
      1    0.0%    0.0%  LazyCompile: *isArrayType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:65667:29
      1    0.0%    0.0%  LazyCompile: *internIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31498:34
      1    0.0%    0.0%  LazyCompile: *instantiateTypeWithSingleGenericCallSignature /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:77562:63
      1    0.0%    0.0%  LazyCompile: *insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      1    0.0%    0.0%  LazyCompile: *initialise /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5131:15
      1    0.0%    0.0%  LazyCompile: *inferFromTypes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:66691:36
      1    0.0%    0.0%  LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8133:25
      1    0.0%    0.0%  LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2177:25
      1    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1    0.0%    0.0%  LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10776:12
      1    0.0%    0.0%  LazyCompile: *hasPostfixPart /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:63:24
      1    0.0%    0.0%  LazyCompile: *hasJSDocNodes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13989:27
      1    0.0%    0.0%  LazyCompile: *hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6530:33
      1    0.0%    0.0%  LazyCompile: *hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6301:33
      1    0.0%    0.0%  LazyCompile: *hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11237:33
      1    0.0%    0.0%  LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7946:15
      1    0.0%    0.0%  LazyCompile: *hasCorrectArity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73394:33
      1    0.0%    0.0%  LazyCompile: *getTypeAtFlowAssignment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:68625:45
      1    0.0%    0.0%  LazyCompile: *getTokenFromCode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:5618:19
      1    0.0%    0.0%  LazyCompile: *getStringLiteralType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:61368:38
      1    0.0%    0.0%  LazyCompile: *getSpreadArgumentIndex /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73385:40
      1    0.0%    0.0%  LazyCompile: *getSourceTextOfNodeFromSourceFile /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:14550:47
      1    0.0%    0.0%  LazyCompile: *getSignaturesOfType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57883:37
      1    0.0%    0.0%  LazyCompile: *getSignatureApplicabilityError /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73767:48
      1    0.0%    0.0%  LazyCompile: *getReducedType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57786:32
      1    0.0%    0.0%  LazyCompile: *getPropertyOfUnionOrIntersectionType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57775:54
      1    0.0%    0.0%  LazyCompile: *getPropertyOfObjectType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57204:41
      1    0.0%    0.0%  LazyCompile: *getPipelinePhase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108420:34
      1    0.0%    0.0%  LazyCompile: *getPathIfNotComputed /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9147:30
      1    0.0%    0.0%  LazyCompile: *getNodePos /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31128:28
      1    0.0%    0.0%  LazyCompile: *getNodeConstructor /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12737:33
      1    0.0%    0.0%  LazyCompile: *getNarrowableTypeForReference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69653:47
      1    0.0%    0.0%  LazyCompile: *getNameOfDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12718:34
      1    0.0%    0.0%  LazyCompile: *getModifierFlagsWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:18608:36
      1    0.0%    0.0%  LazyCompile: *getLiteralValueAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6514:26
      1    0.0%    0.0%  LazyCompile: *getJSDocCommentRanges /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:15255:35
      1    0.0%    0.0%  LazyCompile: *getIsDeferredContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48744:38
      1    0.0%    0.0%  LazyCompile: *getIndexInfosOfStructuredType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57923:47
      1    0.0%    0.0%  LazyCompile: *getExportSymbolOfValueSymbolIfExported /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:50334:56
      1    0.0%    0.0%  LazyCompile: *getEntities /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2113:16
      1    0.0%    0.0%  LazyCompile: *getEffectiveReturnTypeNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:18344:40
      1    0.0%    0.0%  LazyCompile: *getDeclarationModifierFlagsFromSymbol /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:19210:51
      1    0.0%    0.0%  LazyCompile: *getCommaSeparatedNodesWithBoundaries /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5309:46
      1    0.0%    0.0%  LazyCompile: *getBaseConstraintOfType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57394:41
      1    0.0%    0.0%  LazyCompile: *getApparentType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57582:33
      1    0.0%    0.0%  LazyCompile: *get parent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:306:13
      1    0.0%    0.0%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:439:19
      1    0.0%    0.0%  LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:307:17
      1    0.0%    0.0%  LazyCompile: *forEachChildRecursively /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:30421:37
      1    0.0%    0.0%  LazyCompile: *finishToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3969:14
      1    0.0%    0.0%  LazyCompile: *finishNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3837:13
      1    0.0%    0.0%  LazyCompile: *find /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/index.js:47:14
      1    0.0%    0.0%  LazyCompile: *extname node:path:1385:10
      1    0.0%    0.0%  LazyCompile: *enter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3426:8
      1    0.0%    0.0%  LazyCompile: *emitTrailingCommentsOfNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112073:44
      1    0.0%    0.0%  LazyCompile: *emitTokenWithComment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:109984:38
      1    0.0%    0.0%  LazyCompile: *emitSourcePos /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112406:31
      1    0.0%    0.0%  LazyCompile: *emitExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108388:32
      1    0.0%    0.0%  LazyCompile: *emit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108376:22
      1    0.0%    0.0%  LazyCompile: *deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9222:32
      1    0.0%    0.0%  LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6335:19
      1    0.0%    0.0%  LazyCompile: *createScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7941:16
      1    0.0%    0.0%  LazyCompile: *createNodeArray /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31463:33
      1    0.0%    0.0%  LazyCompile: *createNodeArray /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:22042:33
      1    0.0%    0.0%  LazyCompile: *createBaseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:30571:38
      1    0.0%    0.0%  LazyCompile: *coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      1    0.0%    0.0%  LazyCompile: *computeLineStarts /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10081:31
      1    0.0%    0.0%  LazyCompile: *computeLineAndCharacterOfPosition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10149:47
      1    0.0%    0.0%  LazyCompile: *commitPendingMapping /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:87855:38
      1    0.0%    0.0%  LazyCompile: *codePointAt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12116:63
      1    0.0%    0.0%  LazyCompile: *cloneLocation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/source-coverage.js:10:23
      1    0.0%    0.0%  LazyCompile: *checkPropertyAccessExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72563:47
      1    0.0%    0.0%  LazyCompile: *checkPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/lib/virtual-types.js:99:12
      1    0.0%    0.0%  LazyCompile: *checkIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69699:33
      1    0.0%    0.0%  LazyCompile: *bindWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:45526:28
      1    0.0%    0.0%  LazyCompile: *bindCallExpressionFlow /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:44876:40
      1    0.0%    0.0%  LazyCompile: *applyDeoptimizations /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9583:25
      1    0.0%    0.0%  LazyCompile: *appendIfUnique /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:1233:28
      1    0.0%    0.0%  LazyCompile: *append /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:1174:20
      1    0.0%    0.0%  LazyCompile: *addUsedOutsideNames /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6909:24
      1    0.0%    0.0%  LazyCompile: *_splitChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1638:58
      1    0.0%    0.0%  LazyCompile: *_parameters /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/methods.js:27:21
      1    0.0%    0.0%  LazyCompile: *VariableDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8368:1
      1    0.0%    0.0%  LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:64:26
      1    0.0%    0.0%  LazyCompile: *SourceLocation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:62:14
      1    0.0%    0.0%  LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
      1    0.0%    0.0%  LazyCompile: *ReturnValueScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7843:16
      1    0.0%    0.0%  LazyCompile: *ReferencedIdentifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:180:23
      1    0.0%    0.0%  LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:77:26
      1    0.0%    0.0%  LazyCompile: *NodePath.<computed> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/index.js:241:46
      1    0.0%    0.0%  LazyCompile: *NodePath.<computed> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/index.js:225:42
      1    0.0%    0.0%  LazyCompile: *Function /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:253:11
      1    0.0%    0.0%  LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7905:1
      1    0.0%    0.0%  LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:212:29
      1    0.0%    0.0%  LazyCompile: *Declaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:173:14
      1    0.0%    0.0%  LazyCompile: *ChildScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6880:16
      1    0.0%    0.0%  LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9419:16
      1    0.0%    0.0%  LazyCompile: *ArraySet_add /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/node_modules/source-map/lib/array-set.js:49:47
      1    0.0%    0.0%  LazyCompile: *<anonymous> node:fs:188:10
      1    0.0%    0.0%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72881:53
      1    0.0%    0.0%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69570:58
      1    0.0%    0.0%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:216:35
      1    0.0%    0.0%  LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:142:19
      1    0.0%    0.0%  Function: ^writeTokenNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:111375:32
      1    0.0%    0.0%  Function: ^writeKeyword /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:111331:30
      1    0.0%    0.0%  Function: ^widenTypeForVariableLikeDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:54964:53
      1    0.0%    0.0%  Function: ^watchFilePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:120825:31
      1    0.0%    0.0%  Function: ^validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
      1    0.0%    0.0%  Function: ^unwrapAwaitedType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:79200:35
      1    0.0%    0.0%  Function: ^types$1.braceL.updateContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18418:40
      1    0.0%    0.0%  Function: ^tryParseSemicolon /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31450:35
      1    0.0%    0.0%  Function: ^traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
      1    0.0%    0.0%  Function: ^transformES2020 /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:94893:29
      1    0.0%    0.0%  Function: ^trackEntityAtPathAndGetIfTracked /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2097:37
      1    0.0%    0.0%  Function: ^traceSegment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:13958:17
      1    0.0%    0.0%  Function: ^token /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31140:23
      1    0.0%    0.0%  Function: ^toReferencedListDeep /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10913:23
      1    0.0%    0.0%  Function: ^toReferencedList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:5667:19
      1    0.0%    0.0%  Function: ^toAssignableList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10827:19
      1    0.0%    0.0%  Function: ^startsWith /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:2270:24
      1    0.0%    0.0%  Function: ^skipTrivia /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10275:24
      1    0.0%    0.0%  Function: ^skipSpace /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2075:12
      1    0.0%    0.0%  Function: ^skipOuterExpressions /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29150:34
      1    0.0%    0.0%  Function: ^signatureHasLiteralTypes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:86982:38
      1    0.0%    0.0%  Function: ^setUpChunkImportsAndExportsForModule /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:15358:41
      1    0.0%    0.0%  Function: ^setLexicalEnvironmentFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:107250:44
      1    0.0%    0.0%  Function: ^right /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29715:23
      1    0.0%    0.0%  Function: ^result /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/node_modules/@rollup/pluginutils/dist/cjs/index.js:321:27
      1    0.0%    0.0%  Function: ^resolveIdViaPlugins /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21734:29
      1    0.0%    0.0%  Function: ^resolveCallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:74474:39
      1    0.0%    0.0%  Function: ^resolveAnonymousTypeMembers /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:56835:45
      1    0.0%    0.0%  Function: ^resolve node:path:1091:10
      1    0.0%    0.0%  Function: ^replaceWith /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/replacement.js:102:21
      1    0.0%    0.0%  Function: ^renderStatementList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5272:29
      1    0.0%    0.0%  Function: ^render /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7975:11
      1    0.0%    0.0%  Function: ^relative node:path:1190:11
      1    0.0%    0.0%  Function: ^recursiveTypeRelatedTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:63793:44
      1    0.0%    0.0%  Function: ^reassign /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/binding.js:48:11
      1    0.0%    0.0%  Function: ^readToken_eq_excl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2391:20
      1    0.0%    0.0%  Function: ^readTmplToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2920:16
      1    0.0%    0.0%  Function: ^readSync node:fs:684:18
      1    0.0%    0.0%  Function: ^readString /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2889:13
      1    0.0%    0.0%  Function: ^readInt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:2678:10
      1    0.0%    0.0%  Function: ^pushTypeResolution /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:54119:36
      1    0.0%    0.0%  Function: ^pushComment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36643:41
      1    0.0%    0.0%  Function: ^propertiesRelatedTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:64731:41
      1    0.0%    0.0%  Function: ^pp.readToken_eq_excl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21126:32
      1    0.0%    0.0%  Function: ^pp.readRadixNumber /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21349:30
      1    0.0%    0.0%  Function: ^pp.readEscapedChar /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21540:30
      1    0.0%    0.0%  Function: ^pp.getTokenFromCode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21169:31
      1    0.0%    0.0%  Function: ^pp$8.parseVar /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17431:25
      1    0.0%    0.0%  Function: ^pp$8.parseStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:17016:31
      1    0.0%    0.0%  Function: ^pp$5.parseTemplateElement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19135:37
      1    0.0%    0.0%  Function: ^pp$5.parsePropertyName /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19304:34
      1    0.0%    0.0%  Function: ^pp$5.parseMaybeUnary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18687:32
      1    0.0%    0.0%  Function: ^pp$5.parseExprOp /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:18651:28
      1    0.0%    0.0%  Function: ^pp$3.exitScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:19613:26
      1    0.0%    0.0%  Function: ^populatePlaceholders /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/template/lib/populate.js:14:30
      1    0.0%    0.0%  Function: ^pipelineEmitWithSourceMaps /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112340:44
      1    0.0%    0.0%  Function: ^pipelineEmitWithComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112015:42
      1    0.0%    0.0%  Function: ^pipelineEmit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:108404:30
      1    0.0%    0.0%  Function: ^picomatch.test /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-pluginutils/dist/pluginutils.cjs.js:2554:18
      1    0.0%    0.0%  Function: ^picomatch.test /home/matt/Projects/darkreader_ws/_darkreader/node_modules/picomatch/lib/picomatch.js:117:18
      1    0.0%    0.0%  Function: ^pathComponents /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:6202:28
      1    0.0%    0.0%  Function: ^parseVarStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:13767:20
      1    0.0%    0.0%  Function: ^parseUpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:34132:39
      1    0.0%    0.0%  Function: ^parseUnionOrIntersectionType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:33263:46
      1    0.0%    0.0%  Function: ^parseTrailingTagComments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:36872:50
      1    0.0%    0.0%  Function: ^parseTopLevel /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:6258:16
      1    0.0%    0.0%  Function: ^parsePropertyDefinition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12470:26
      1    0.0%    0.0%  Function: ^parseParenItem /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:5480:17
      1    0.0%    0.0%  Function: ^parseObjectTypeMembers /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:32889:40
      1    0.0%    0.0%  Function: ^parseObjectLike /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12411:18
      1    0.0%    0.0%  Function: ^parseObjPropValue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:5815:20
      1    0.0%    0.0%  Function: ^parseNewExpressionOrNewDotTarget /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:34922:50
      1    0.0%    0.0%  Function: ^parseMember /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11598:14
      1    0.0%    0.0%  Function: ^parseLeftHandSideExpressionOrHigher /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:34150:53
      1    0.0%    0.0%  Function: ^parseImport /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:14727:14
      1    0.0%    0.0%  Function: ^parseFunctionDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:35677:42
      1    0.0%    0.0%  Function: ^parseExprOps /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11313:15
      1    0.0%    0.0%  Function: ^parseExprOp /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11326:14
      1    0.0%    0.0%  Function: ^parseExprListItem /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12791:20
      1    0.0%    0.0%  Function: ^parseExprAtom /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:7454:16
      1    0.0%    0.0%  Function: ^parseContextualModifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31597:41
      1    0.0%    0.0%  Function: ^parseCallExpressionRest /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:34642:41
      1    0.0%    0.0%  Function: ^parseBindingList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10954:19
      1    0.0%    0.0%  Function: ^parseArrowExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12682:23
      1    0.0%    0.0%  Function: ^parseAnyContextualModifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31642:44
      1    0.0%    0.0%  Function: ^parse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/picomatch/lib/parse.js:55:15
      1    0.0%    0.0%  Function: ^onExit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:76884:28
      1    0.0%    0.0%  Function: ^onEnter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:107:10
      1    0.0%    0.0%  Function: ^nodeIsMissing /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:14403:27
      1    0.0%    0.0%  Function: ^nextToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:31150:27
      1    0.0%    0.0%  Function: ^narrowTypeByInstanceof /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69384:44
      1    0.0%    0.0%  Function: ^narrowTypeByBinaryExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69014:50
      1    0.0%    0.0%  Function: ^narrowType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69488:32
      1    0.0%    0.0%  Function: ^modifiersToFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:18684:30
      1    0.0%    0.0%  Function: ^maybeParseModuleAttributes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:14808:29
      1    0.0%    0.0%  Function: ^maybeBind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:44721:31
      1    0.0%    0.0%  Function: ^markNodeAssignments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:69597:37
      1    0.0%    0.0%  Function: ^makeSynthetic /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:26634:27
      1    0.0%    0.0%  Function: ^length /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:379:20
      1    0.0%    0.0%  Function: ^left /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29683:22
      1    0.0%    0.0%  Function: ^is_reference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5415:23
      1    0.0%    0.0%  Function: ^isValidTypeOnlyAliasUseSite /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:20675:41
      1    0.0%    0.0%  Function: ^isUnparsedNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13139:28
      1    0.0%    0.0%  Function: ^isURLInstance node:internal/url:1492:23
      1    0.0%    0.0%  Function: ^isTypeOrBaseIdenticalTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:67320:41
      1    0.0%    0.0%  Function: ^isThislessInterface /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:55792:37
      1    0.0%    0.0%  Function: ^isStartOfLeftHandSideExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:33394:49
      1    0.0%    0.0%  Function: ^isSpreadAssignment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:28612:32
      1    0.0%    0.0%  Function: ^isPropertyAccessEntityNameExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:18806:50
      1    0.0%    0.0%  Function: ^isPartialMappedType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57160:37
      1    0.0%    0.0%  Function: ^isNodeType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/introspection.js:58:20
      1    0.0%    0.0%  Function: ^isMethodAccessForCall /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72576:39
      1    0.0%    0.0%  Function: ^isMatchingReference /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:67527:37
      1    0.0%    0.0%  Function: ^isLogicalAssignmentExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:44224:47
      1    0.0%    0.0%  Function: ^isIdentifierChar /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:16229:26
      1    0.0%    0.0%  Function: ^isGenericIndexType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:60636:36
      1    0.0%    0.0%  Function: ^isFunctionOrModuleBlock /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:13368:37
      1    0.0%    0.0%  Function: ^isFunctionDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/validators/generated/index.js:588:31
      1    0.0%    0.0%  Function: ^isContextual /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:3478:15
      1    0.0%    0.0%  Function: ^isConflictMarkerTrivia /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:10368:36
      1    0.0%    0.0%  Function: ^isBlockScopedNameDeclaredBeforeUse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:48058:52
      1    0.0%    0.0%  Function: ^isBinaryCheck /home/matt/Projects/darkreader_ws/_darkreader/node_modules/isbinaryfile/lib/index.js:67:23
      1    0.0%    0.0%  Function: ^instantiateTypeWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:61971:39
      1    0.0%    0.0%  Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
      1    0.0%    0.0%  Function: ^inferFromTypes /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:66691:36
      1    0.0%    0.0%  Function: ^includeVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:13058:20
      1    0.0%    0.0%  Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1    0.0%    0.0%  Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      1    0.0%    0.0%  Function: ^hookFirst /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22661:14
      1    0.0%    0.0%  Function: ^hasPlugin /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10585:19
      1    0.0%    0.0%  Function: ^hasMatchingArgument /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:67729:37
      1    0.0%    0.0%  Function: ^hasEffectsWhenCalledAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8077:31
      1    0.0%    0.0%  Function: ^hasEffectsWhenAssignedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6387:33
      1    0.0%    0.0%  Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9522:33
      1    0.0%    0.0%  Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7214:33
      1    0.0%    0.0%  Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6301:33
      1    0.0%    0.0%  Function: ^hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9930:15
      1    0.0%    0.0%  Function: ^hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11760:15
      1    0.0%    0.0%  Function: ^hasCorrectTypeArgumentArity /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73461:45
      1    0.0%    0.0%  Function: ^handleExistingModule /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22356:31
      1    0.0%    0.0%  Function: ^getTypeReferenceType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:58824:38
      1    0.0%    0.0%  Function: ^getTypePredicateOfSignature /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:58240:45
      1    0.0%    0.0%  Function: ^getTypeOfSymbolWithDeferredType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:55343:49
      1    0.0%    0.0%  Function: ^getTypeAtPosition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:75513:35
      1    0.0%    0.0%  Function: ^getTextOfNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:111619:31
      1    0.0%    0.0%  Function: ^getTestValue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10207:17
      1    0.0%    0.0%  Function: ^getStatsFromBinding node:internal/fs/utils:530:29
      1    0.0%    0.0%  Function: ^getSpreadArgumentIndex /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73385:40
      1    0.0%    0.0%  Function: ^getSimplifiedType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:60660:35
      1    0.0%    0.0%  Function: ^getSelectedEffectiveModifierFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:18600:47
      1    0.0%    0.0%  Function: ^getReturnTypeFromBody /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:75780:39
      1    0.0%    0.0%  Function: ^getReturnExpressionWhenCalledAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6371:40
      1    0.0%    0.0%  Function: ^getRegularTypeOfLiteralType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:61360:45
      1    0.0%    0.0%  Function: ^getPropertyKey /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9387:19
      1    0.0%    0.0%  Function: ^getPropertiesOfUnionOrIntersectionType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57213:56
      1    0.0%    0.0%  Function: ^getPropertiesOfType /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:57237:37
      1    0.0%    0.0%  Function: ^getOutputs /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:107503:28
      1    0.0%    0.0%  Function: ^getOptions /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10721:20
      1    0.0%    0.0%  Function: ^getObjectTypeInstantiation /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:61728:44
      1    0.0%    0.0%  Function: ^getDiscriminantPropertyAccess /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:68935:51
      1    0.0%    0.0%  Function: ^getDirectoryPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:6140:30
      1    0.0%    0.0%  Function: ^getDeclaredTypeOfClassOrInterface /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:55818:51
      1    0.0%    0.0%  Function: ^getContainerFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:44904:35
      1    0.0%    0.0%  Function: ^getCombinedFlags /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12431:30
      1    0.0%    0.0%  Function: ^getAssignmentDeclarationPropertyAccessKind /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:16384:56
      1    0.0%    0.0%  Function: ^getActualTypeVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:60875:39
      1    0.0%    0.0%  Function: ^findTypescriptOutput /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-typescript/dist/index.js:442:30
      1    0.0%    0.0%  Function: ^findFirstLineBreakOutsideComment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5255:42
      1    0.0%    0.0%  Function: ^fetchModuleDependencies /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22259:34
      1    0.0%    0.0%  Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:29733:22
      1    0.0%    0.0%  Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
      1    0.0%    0.0%  Function: ^endsWith /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:145:11
      1    0.0%    0.0%  Function: ^emitTrailingCommentsOfPosition /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112247:48
      1    0.0%    0.0%  Function: ^emitSourceMapsAfterNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112369:41
      1    0.0%    0.0%  Function: ^emitPropertyAssignment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:110594:40
      1    0.0%    0.0%  Function: ^emitLeadingCommentsOfNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:112042:43
      1    0.0%    0.0%  Function: ^dirname node:path:1275:10
      1    0.0%    0.0%  Function: ^deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9222:32
      1    0.0%    0.0%  Function: ^deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7124:32
      1    0.0%    0.0%  Function: ^deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6511:32
      1    0.0%    0.0%  Function: ^deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2151:32
      1    0.0%    0.0%  Function: ^deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9463:19
      1    0.0%    0.0%  Function: ^declareSymbolAndAddToSymbolTable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:44981:50
      1    0.0%    0.0%  Function: ^declareName /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:1762:14
      1    0.0%    0.0%  Function: ^createScope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8022:16
      1    0.0%    0.0%  Function: ^createLiteralLikeNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:22236:39
      1    0.0%    0.0%  Function: ^createInclusionContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:4586:32
      1    0.0%    0.0%  Function: ^createHasEffectsContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:4593:33
      1    0.0%    0.0%  Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1    0.0%    0.0%  Function: ^codePointAt /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:12116:63
      1    0.0%    0.0%  Function: ^checkVariableLikeDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:80357:46
      1    0.0%    0.0%  Function: ^checkUnreachable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:46383:34
      1    0.0%    0.0%  Function: ^checkTypeRelatedTo /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:62987:36
      1    0.0%    0.0%  Function: ^checkStrictModeBinaryExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:45303:49
      1    0.0%    0.0%  Function: ^checkParams /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:12757:14
      1    0.0%    0.0%  Function: ^checkObjectLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:71577:36
      1    0.0%    0.0%  Function: ^checkNonNullTypeWithReporter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:72540:46
      1    0.0%    0.0%  Function: ^checkGrammarObjectLiteralExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:85980:53
      1    0.0%    0.0%  Function: ^checkExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:77780:33
      1    0.0%    0.0%  Function: ^checkExponentialAfterUnary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:11446:29
      1    0.0%    0.0%  Function: ^checkCollisionsForDeclarationName /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:80271:51
      1    0.0%    0.0%  Function: ^checkCallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:75103:37
      1    0.0%    0.0%  Function: ^checkBinaryLikeExpressionWorker /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:76957:49
      1    0.0%    0.0%  Function: ^bindPropertyOrMethodOrAccessor /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:46334:48
      1    0.0%    0.0%  Function: ^bindChildren /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:43922:30
      1    0.0%    0.0%  Function: ^assertIsDefined /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:2601:33
      1    0.0%    0.0%  Function: ^applyReplacement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/template/lib/populate.js:45:26
      1    0.0%    0.0%  Function: ^applyDeoptimizations /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9583:25
      1    0.0%    0.0%  Function: ^appendIfCommentDirective /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:11775:42
      1    0.0%    0.0%  Function: ^allocate node:buffer:397:18
      1    0.0%    0.0%  Function: ^addUsedOutsideNames /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6909:24
      1    0.0%    0.0%  Function: ^addTypeToUnion /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:59530:32
      1    0.0%    0.0%  Function: ^addJSDocComment /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:30876:33
      1    0.0%    0.0%  Function: ^addInheritedMembers /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:56110:37
      1    0.0%    0.0%  Function: ^_replaceWith /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/replacement.js:163:22
      1    0.0%    0.0%  Function: ^_maybeAddParen /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:173:17
      1    0.0%    0.0%  Function: ^_functionHead /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/methods.js:93:23
      1    0.0%    0.0%  Function: ^_append /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:101:10
      1    0.0%    0.0%  Function: ^__spreadArray /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:17:62
      1    0.0%    0.0%  Function: ^VariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11802:1
      1    0.0%    0.0%  Function: ^UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:64:26
      1    0.0%    0.0%  Function: ^UnaryExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:230:18
      1    0.0%    0.0%  Function: ^ThisVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7683:16
      1    0.0%    0.0%  Function: ^StatWatcher.<computed> node:internal/fs/watchers:99:54
      1    0.0%    0.0%  Function: ^Scope /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:275:14
      1    0.0%    0.0%  Function: ^Position /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:53:14
      1    0.0%    0.0%  Function: ^Module._nodeModulePaths node:internal/modules/cjs/loader:629:37
      1    0.0%    0.0%  Function: ^Minimatch.matchOne /home/matt/Projects/darkreader_ws/_darkreader/node_modules/minimatch/minimatch.js:760:41
      1    0.0%    0.0%  Function: ^MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
      1    0.0%    0.0%  Function: ^MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
      1    0.0%    0.0%  Function: ^MappingList_add /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/node_modules/source-map/lib/mapping-list.js:52:53
      1    0.0%    0.0%  Function: ^LocalVariable /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6310:16
      1    0.0%    0.0%  Function: ^LValParser /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/parser/lib/index.js:10735:1
      1    0.0%    0.0%  Function: ^AwaitExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9019:16
      1    0.0%    0.0%  Function: ^ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      1    0.0%    0.0%  Function: ^<anonymous> node:internal/fs/utils:680:42
      1    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/typescript/lib/typescript.js:73535:106
      1    0.0%    0.0%  Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21959:98

 [C++]:
   ticks  total  nonlib   name
    554    1.7%    7.3%  __futex_abstimed_wait_common
    173    0.5%    2.3%  __memcpy_evex_unaligned_erms
    116    0.4%    1.5%  __GI___libc_write
     99    0.3%    1.3%  _int_malloc
     84    0.3%    1.1%  __GI___mprotect
     84    0.3%    1.1%  __GI___libc_malloc
     76    0.2%    1.0%  fwrite
     71    0.2%    0.9%  __GI___mmap
     67    0.2%    0.9%  _int_free
     62    0.2%    0.8%  __GI___pthread_mutex_lock
     53    0.2%    0.7%  __GI___lll_lock_wait
     27    0.1%    0.4%  unlink_chunk.constprop.0
     26    0.1%    0.3%  cfree@GLIBC_2.2.5
     26    0.1%    0.3%  __GI___munmap
     24    0.1%    0.3%  __GI___lll_lock_wake
     21    0.1%    0.3%  __GI___pthread_cond_signal
     18    0.1%    0.2%  __GI___pthread_mutex_unlock_usercnt
     16    0.0%    0.2%  malloc_consolidate
     14    0.0%    0.2%  __memset_evex_unaligned_erms
     13    0.0%    0.2%  __memchr_evex
     12    0.0%    0.2%  std::basic_ostream<char, std::char_traits<char> >& std::__ostream_insert<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*, long)
     10    0.0%    0.1%  std::ostream::sentry::sentry(std::ostream&)
      9    0.0%    0.1%  _dl_addr
      9    0.0%    0.1%  __GI___printf_fp_l
      9    0.0%    0.1%  __GI___lll_lock_wait_private
      8    0.0%    0.1%  std::_Rb_tree_insert_and_rebalance(bool, std::_Rb_tree_node_base*, std::_Rb_tree_node_base*, std::_Rb_tree_node_base&)
      8    0.0%    0.1%  __GI___pthread_rwlock_unlock
      8    0.0%    0.1%  __GI__IO_file_xsputn
      7    0.0%    0.1%  std::ostream& std::ostream::_M_insert<long>(long)
      7    0.0%    0.1%  __GI___lll_lock_wake_private
      6    0.0%    0.1%  int std::__int_to_char<char, unsigned long>(char*, unsigned long, char const*, std::_Ios_Fmtflags, bool)
      6    0.0%    0.1%  __vfprintf_internal
      6    0.0%    0.1%  __GI___pthread_rwlock_wrlock
      5    0.0%    0.1%  std::ostreambuf_iterator<char, std::char_traits<char> > std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::_M_insert_int<long>(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, long) const
      5    0.0%    0.1%  operator new(unsigned long)
      4    0.0%    0.1%  std::__detail::_Prime_rehash_policy::_M_next_bkt(unsigned long) const
      4    0.0%    0.1%  __strlen_evex
      4    0.0%    0.1%  __memcmp_evex_movbe
      4    0.0%    0.1%  __getrusage
      3    0.0%    0.0%  std::ostream& std::ostream::_M_insert<unsigned long>(unsigned long)
      3    0.0%    0.0%  __clock_gettime
      2    0.0%    0.0%  std::ostreambuf_iterator<char, std::char_traits<char> > std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::_M_insert_int<unsigned long>(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, unsigned long) const
      2    0.0%    0.0%  std::_Rb_tree_rebalance_for_erase(std::_Rb_tree_node_base*, std::_Rb_tree_node_base&)
      2    0.0%    0.0%  std::_Rb_tree_increment(std::_Rb_tree_node_base*)
      2    0.0%    0.0%  operator new[](unsigned long, std::nothrow_t const&)
      2    0.0%    0.0%  operator delete(void*)
      2    0.0%    0.0%  __GI___pthread_getspecific
      2    0.0%    0.0%  __GI__IO_file_sync
      1    0.0%    0.0%  syscall
      1    0.0%    0.0%  std::ostream::flush()
      1    0.0%    0.0%  std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::do_put(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, long) const
      1    0.0%    0.0%  std::locale::id::_M_id() const
      1    0.0%    0.0%  std::__detail::_Prime_rehash_policy::_M_need_rehash(unsigned long, unsigned long, unsigned long) const
      1    0.0%    0.0%  std::_Rb_tree_increment(std::_Rb_tree_node_base const*)
      1    0.0%    0.0%  operator new[](unsigned long)
      1    0.0%    0.0%  new_do_write
      1    0.0%    0.0%  fputc
      1    0.0%    0.0%  fflush
      1    0.0%    0.0%  __strcmp_evex
      1    0.0%    0.0%  __mpn_mul_1
      1    0.0%    0.0%  __mempcpy_evex_unaligned_erms
      1    0.0%    0.0%  __getpid
      1    0.0%    0.0%  __fmod_finite@GLIBC_2.15
      1    0.0%    0.0%  ___vsnprintf
      1    0.0%    0.0%  __GI___pthread_cond_wait
      1    0.0%    0.0%  __GI___pthread_cleanup_push
      1    0.0%    0.0%  __GI___madvise
      1    0.0%    0.0%  _IO_file_write@@GLIBC_2.2.5

 [Summary]:
   ticks  total  nonlib   name
   5713   17.4%   75.7%  JavaScript
   1795    5.5%   23.8%  C++
  11320   34.5%  149.9%  GC
  25298   77.0%          Shared libraries
     42    0.1%          Unaccounted

 [C++ entry points]:
   ticks    cpp   total   name
     79   13.3%    0.2%  __memcpy_evex_unaligned_erms
     73   12.2%    0.2%  __GI___libc_write
     69   11.6%    0.2%  fwrite
     63   10.6%    0.2%  _int_malloc
     55    9.2%    0.2%  __GI___libc_malloc
     35    5.9%    0.1%  __GI___mmap
     24    4.0%    0.1%  __GI___mprotect
     14    2.3%    0.0%  __memset_evex_unaligned_erms
     13    2.2%    0.0%  __memchr_evex
     12    2.0%    0.0%  std::basic_ostream<char, std::char_traits<char> >& std::__ostream_insert<char, std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*, long)
     12    2.0%    0.0%  _int_free
     11    1.8%    0.0%  cfree@GLIBC_2.2.5
     11    1.8%    0.0%  __GI___munmap
     10    1.7%    0.0%  malloc_consolidate
     10    1.7%    0.0%  __GI___pthread_mutex_lock
      9    1.5%    0.0%  unlink_chunk.constprop.0
      8    1.3%    0.0%  std::_Rb_tree_insert_and_rebalance(bool, std::_Rb_tree_node_base*, std::_Rb_tree_node_base*, std::_Rb_tree_node_base&)
      7    1.2%    0.0%  __GI__IO_file_xsputn
      6    1.0%    0.0%  std::ostream::sentry::sentry(std::ostream&)
      6    1.0%    0.0%  std::ostream& std::ostream::_M_insert<long>(long)
      6    1.0%    0.0%  int std::__int_to_char<char, unsigned long>(char*, unsigned long, char const*, std::_Ios_Fmtflags, bool)
      5    0.8%    0.0%  std::ostreambuf_iterator<char, std::char_traits<char> > std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::_M_insert_int<long>(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, long) const
      5    0.8%    0.0%  operator new(unsigned long)
      5    0.8%    0.0%  __GI___pthread_mutex_unlock_usercnt
      5    0.8%    0.0%  __GI___pthread_cond_signal
      4    0.7%    0.0%  __memcmp_evex_movbe
      4    0.7%    0.0%  __GI___pthread_rwlock_wrlock
      4    0.7%    0.0%  __GI___pthread_rwlock_unlock
      3    0.5%    0.0%  std::__detail::_Prime_rehash_policy::_M_next_bkt(unsigned long) const
      3    0.5%    0.0%  __strlen_evex
      2    0.3%    0.0%  std::ostreambuf_iterator<char, std::char_traits<char> > std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::_M_insert_int<unsigned long>(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, unsigned long) const
      2    0.3%    0.0%  std::ostream& std::ostream::_M_insert<unsigned long>(unsigned long)
      2    0.3%    0.0%  std::_Rb_tree_rebalance_for_erase(std::_Rb_tree_node_base*, std::_Rb_tree_node_base&)
      2    0.3%    0.0%  std::_Rb_tree_increment(std::_Rb_tree_node_base*)
      2    0.3%    0.0%  operator new[](unsigned long, std::nothrow_t const&)
      2    0.3%    0.0%  __GI___pthread_getspecific
      2    0.3%    0.0%  __GI__IO_file_sync
      1    0.2%    0.0%  syscall
      1    0.2%    0.0%  std::num_put<char, std::ostreambuf_iterator<char, std::char_traits<char> > >::do_put(std::ostreambuf_iterator<char, std::char_traits<char> >, std::ios_base&, char, long) const
      1    0.2%    0.0%  std::locale::id::_M_id() const
      1    0.2%    0.0%  std::_Rb_tree_increment(std::_Rb_tree_node_base const*)
      1    0.2%    0.0%  operator new[](unsigned long)
      1    0.2%    0.0%  operator delete(void*)
      1    0.2%    0.0%  new_do_write
      1    0.2%    0.0%  __strcmp_evex
      1    0.2%    0.0%  __fmod_finite@GLIBC_2.15
      1    0.2%    0.0%  __clock_gettime
      1    0.2%    0.0%  __GI___lll_lock_wait

 [Bottom up (heavy) profile]:
  Note: percentage shows a share of a particular caller in the total
  amount of its parent calls.
  Callers occupying less than 1.0% are not shown.

   ticks parent  name
  25296   77.0%  /home/matt/.nvm/versions/node/v16.14.0/bin/node
   6175   24.4%    /home/matt/.nvm/versions/node/v16.14.0/bin/node
   1045   16.9%      /home/matt/.nvm/versions/node/v16.14.0/bin/node
    262   25.1%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
    253   96.6%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8288:14
    253  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      5    1.9%          Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8288:14
      4   80.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      1   20.0%            Function: ^NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      4    1.5%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      2   50.0%            LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6457:16
      1   25.0%            LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
      1   25.0%            Function: ^AssignmentExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8902:16
    141   13.5%        LazyCompile: *deoptimizeAllProperties /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7063:28
    132   93.6%          LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7092:19
    108   81.8%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11048:19
     11    8.3%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6978:19
     10    7.6%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8664:19
      3    2.3%            Function: ^deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8664:19
      9    6.4%          Function: ^deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7092:19
      3   33.3%            Function: ^deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11048:19
      2   22.2%            LazyCompile: *deoptimizeAllProperties /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7063:28
      2   22.2%            Function: ^deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8664:19
      1   11.1%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6978:19
      1   11.1%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11048:19
     88    8.4%        LazyCompile: *shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7923:21
     43   48.9%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
     17   39.5%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     12   27.9%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
     10   23.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      3    7.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1    2.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
     22   25.0%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     14   63.6%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      3   13.6%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      2    9.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      2    9.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      1    4.5%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
     16   18.2%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      9   56.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      3   18.8%            Function: ^includeAsSingleStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5125:29
      3   18.8%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1    6.3%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      7    8.0%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      3   42.9%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9324:25
      2   28.6%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7800:25
      1   14.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
      1   14.3%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     85    8.1%        LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
     84   98.8%          LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     84  100.0%            LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
      1    1.2%          Function: ^traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:42:18
      1  100.0%            Function: ^instrumentSync /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:107:17
     79    7.6%        LazyCompile: *shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5183:21
     26   32.9%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
     26  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
     22   27.8%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     10   45.5%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      7   31.8%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9534:12
      3   13.6%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      2    9.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
     12   15.2%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     12  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      9   11.4%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      9  100.0%            Function: ^includeStatements /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22968:22
      4    5.1%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      3   75.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      1   25.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      3    3.8%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      2   66.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1   33.3%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1    1.3%          Function: ^includeUnknownTest /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10231:23
      1  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      1    1.3%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      1    1.3%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
      1  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
     54    5.2%        LazyCompile: *createHasEffectsContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:4593:33
     18   33.3%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      7   38.9%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      4   22.2%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      4   22.2%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      3   16.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
     10   18.5%          LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7800:25
     10  100.0%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8133:25
      9   16.7%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      5   55.6%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      2   22.2%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1   11.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      1   11.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      9   16.7%          Function: ^shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5183:21
      3   33.3%            Function: ^shouldBeIncluded /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7923:21
      3   33.3%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1   11.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      1   11.1%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1   11.1%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11247:12
      6   11.1%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8916:12
      3   50.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      2   33.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11200:12
      1   16.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10776:12
      2    3.7%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8916:12
      2  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
     39    3.7%        LazyCompile: *hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6530:33
     37   94.9%          LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9261:15
     32   86.5%            LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9261:15
      3    8.1%            LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9505:15
      1    2.7%            LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5088:15
      1    2.7%            Function: ^hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10116:15
      2    5.1%          Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9276:33
      1   50.0%            Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11237:33
      1   50.0%            Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10752:33
     24    2.3%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11247:12
      9   37.5%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
      8   88.9%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1   11.1%            Function: ^includeAsSingleStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8384:29
      4   16.7%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1   25.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1   25.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1   25.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      1   25.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      4   16.7%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9808:12
      2   50.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      1   25.0%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7800:25
      1   25.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11247:12
      3   12.5%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10776:12
      2   66.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      1   33.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      3   12.5%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
      2   66.7%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1   33.3%            Function: ^includeAsSingleStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8384:29
      1    4.2%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10776:12
      1  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
     16    1.5%        LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     16  100.0%          LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
     13   81.3%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
      2   12.5%            LazyCompile: *crawl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:709:8
      1    6.3%            Function: ^traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:42:18
     15    1.4%        LazyCompile: *trackEntityAtPathAndGetIfTracked /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:2097:37
     10   66.7%          LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9210:19
      7   70.0%            LazyCompile: *deoptimizePath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9210:19
      2   20.0%            LazyCompile: *deoptimizeThisOnEventAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9473:32
      1   10.0%            LazyCompile: *hasEffects /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9505:15
      5   33.3%          Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6380:33
      5  100.0%            Function: ^hasEffectsWhenAccessedAtPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6530:33
     13    1.2%        Function: ^guessIndent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:198:21
     13  100.0%          Function: ^MagicString /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:351:39
     13  100.0%            Function: ^executeReplacement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:86:30
     12    1.1%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      7   58.3%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      4   57.1%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
      1   14.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
      1   14.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      1   14.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      2   16.7%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      1   50.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1   50.0%            Function: ^includeAsSingleStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5125:29
      2   16.7%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:12625:12
      2  100.0%            Function: ^includeStatements /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22968:22
      1    8.3%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10139:12
      1  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     11    1.1%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     11  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      9   81.8%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      2   18.2%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
     11    1.1%        LazyCompile: *pp.readNumber /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21363:25
     11  100.0%          LazyCompile: *pp.getTokenFromCode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21169:31
     11  100.0%            LazyCompile: *pp.readToken /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:20923:24
    774   12.5%      LazyCompile: *decode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:632:16
    774  100.0%        Function: ^decodedSourcemap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:14305:26
    765   98.8%          LazyCompile: ~renderChunkReducer /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:14330:32
    765  100.0%            Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22703:41
      8    1.0%          Function: ^transformReducer /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21928:30
      8  100.0%            Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22703:41
    489    7.9%      Function: ^compileFunction node:vm:308:25
    485   99.2%        Function: ^wrapSafe node:internal/modules/cjs/loader:1017:18
    482   99.4%          Function: ^Module._compile node:internal/modules/cjs/loader:1057:37
    477   99.0%            Function: ^Module._extensions..js node:internal/modules/cjs/loader:1112:37
      5    1.0%            LazyCompile: ~Module._extensions..js node:internal/modules/cjs/loader:1112:37
    342    5.5%      LazyCompile: *indent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1249:48
    342  100.0%        Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1953:32
    342  100.0%          /home/matt/.nvm/versions/node/v16.14.0/bin/node
    342  100.0%            LazyCompile: ~indent /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:1942:45
    288    4.7%      LazyCompile: *encode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/sourcemap-codec/dist/sourcemap-codec.umd.js:78:17
    288  100.0%        Function: ^SourceMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:181:35
    288  100.0%          Function: ^generateMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:497:58
    288  100.0%            Function: ^executeReplacement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:86:30
    223    3.6%      LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:379:11
    107   48.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:122:20
     87   81.3%          LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:627:36
     87  100.0%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     11   10.3%          LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:523:36
     11  100.0%            LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
      9    8.4%          LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
      8   88.9%            LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      1   11.1%            Function: ^memberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:378:26
     95   42.6%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     94   98.9%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     94  100.0%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      1    1.1%          Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
      1  100.0%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:131:13
     21    9.4%        LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
     21  100.0%          LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
     10   47.6%            LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      3   14.3%            LazyCompile: *insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      3   14.3%            LazyCompile: *coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      2    9.5%            LazyCompile: *getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      2    9.5%            Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      1    4.8%            Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
    140    2.3%      LazyCompile: *fields.Object.assign.name.validate.Object.assign.type /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:363:97
    140  100.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
    140  100.0%          LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
    107   76.4%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     33   23.6%            LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
    120    1.9%      LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:95:21
    120  100.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
    119   99.2%          LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
     99   83.2%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      6    5.0%            LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      6    5.0%            Function: ^sequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:410:28
      4    3.4%            LazyCompile: *arrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:258:25
      2    1.7%            Function: ^blockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:282:24
      2    1.7%            Function: ^arrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:258:25
    117    1.9%      LazyCompile: *addUneditedChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:299:65
    117  100.0%        Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:469:36
    117  100.0%          Function: ^eachNext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:60:46
     73   62.4%            Function: ^generateDecodedMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:454:72
     44   37.6%            LazyCompile: ~generateDecodedMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:454:72
    115    1.9%      LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:667:21
    115  100.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:122:20
    115  100.0%          LazyCompile: *validator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:95:21
    115  100.0%            LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
    111    1.8%      LazyCompile: *<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:648:20
    111  100.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
    111  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    110   99.1%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    110    1.8%      LazyCompile: *fields.shorthand.validate.Object.assign.type /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/core.js:640:98
    110  100.0%        LazyCompile: *validate /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/definitions/utils.js:225:20
    110  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    109   99.1%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     96    1.6%      LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
     66   68.8%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     66  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     66  100.0%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     19   19.8%        LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      8   42.1%          LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      5   62.5%            LazyCompile: *coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      2   25.0%            Function: ^coverVariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:349:33
      1   12.5%            Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      6   31.6%          LazyCompile: *insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      4   66.7%            Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1   16.7%            LazyCompile: *coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1   16.7%            Function: ^coverAssignmentPattern /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:339:32
      4   21.1%          Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
      4  100.0%            Function: ^coverFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:345:23
      1    5.3%          Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1  100.0%            Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      5    5.2%        LazyCompile: *wrap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:173:5
      5  100.0%          LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      2   40.0%            LazyCompile: *insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      2   40.0%            LazyCompile: *getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1   20.0%            LazyCompile: *coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      4    4.2%        Function: ^numericLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:358:24
      3   75.0%          Function: ^increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      2   66.7%            Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1   33.3%            Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      1   25.0%          Function: ^valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      1  100.0%            Function: ^valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      2    2.1%        LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      2  100.0%          Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      1   50.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1   50.0%            Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:318:19
    721    2.9%    LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
    673   93.3%      LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
    622   92.4%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
    586   94.2%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
    512   87.4%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5106:12
     39    6.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     22    3.8%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
     12    2.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     19    3.1%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     18   94.7%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      1    5.3%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
     10    1.6%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      8   80.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      2   20.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
     17    2.5%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11814:12
     15   88.2%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
     15  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      2   11.8%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      2  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
     11    1.6%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8377:12
      9   81.8%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      9  100.0%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9534:12
      2   18.2%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      2  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      9    1.3%        LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7957:12
      3   33.3%          LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      1   33.3%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9324:25
      1   33.3%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6554:25
      1   33.3%            LazyCompile: *include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9534:12
      3   33.3%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8827:12
      3  100.0%            LazyCompile: *includeCallArguments /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9324:25
      2   22.2%          Function: ^includeAsSingleStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5125:29
      1   50.0%            Function: ^includeUnknownTest /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10231:23
      1   50.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10063:12
      1   11.1%          Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8118:12
      1  100.0%            Function: ^include /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8316:12
    527    2.1%    LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
    437   82.9%      LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
    212   48.5%        LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
    209   98.6%          LazyCompile: *Property /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11122:16
    209  100.0%            LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     67   15.3%        LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6457:16
     67  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     66   98.5%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      1    1.5%            Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8143:14
     57   13.0%        LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11041:16
     57  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     57  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     28    6.4%        LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
     28  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     28  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     12    2.7%        LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11756:16
     12  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     12  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     10    2.3%        LazyCompile: *BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7931:16
      4   40.0%          Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8851:14
      4  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      3   30.0%          Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10154:14
      3  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      2   20.0%          Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8143:14
      2  100.0%            Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8157:14
      1   10.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      1  100.0%            Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9623:14
      5    1.1%        LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7905:1
      5  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      5  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      5    1.1%        Function: ^SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
      5  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      5  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     81   15.4%      LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8288:14
     81  100.0%        LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     81  100.0%          LazyCompile: *Literal /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8252:1
     81  100.0%            LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
    413    1.6%    LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    388   93.9%      LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    320   82.5%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    239   74.7%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    134   56.1%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    105   43.9%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
     80   25.0%          Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
     80  100.0%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:131:13
     68   17.5%        Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
     68  100.0%          Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:131:13
     68  100.0%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     25    6.1%      Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
     25  100.0%        Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:131:13
     25  100.0%          LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     25  100.0%            LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    362    1.4%    LazyCompile: *builder /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/builder.js:12:17
    205   56.6%      LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    202   98.5%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    201   99.5%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
    169   84.1%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     32   15.9%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
      3    1.5%        Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
      3  100.0%          Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/instrumenter.js:131:13
      3  100.0%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
     75   20.7%      LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
     36   48.0%        LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
     14   38.9%          LazyCompile: *coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
     14  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
     12   33.3%          Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
     12  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
     10   27.8%          Function: ^coverVariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:349:33
     10  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
     14   18.7%        Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
     14  100.0%          Function: ^coverFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:345:23
     13   92.9%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1    7.1%            Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:318:19
     13   17.3%        LazyCompile: *insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      7   53.8%          Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      7  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      5   38.5%          LazyCompile: *coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      5  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1    7.7%          Function: ^coverTernary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:448:22
      1  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      7    9.3%        LazyCompile: *getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      4   57.1%          Function: ^coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      4  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      3   42.9%          Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      2   66.7%            Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1   33.3%            Function: ^coverTernary /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:448:22
      2    2.7%        Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      2  100.0%          Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      2  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      2    2.7%        Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1   50.0%          Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      1  100.0%            Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1   50.0%          Function: ^coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      1  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1    1.3%        LazyCompile: *coverLogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:463:32
      1  100.0%          Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:318:19
      1  100.0%            /home/matt/.nvm/versions/node/v16.14.0/bin/node
     21    5.8%      LazyCompile: *stringLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:354:23
     21  100.0%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     21  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     20   95.2%            Function: ^exit /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:614:9
      1    4.8%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
     15    4.1%      LazyCompile: *insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      6   40.0%        Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      4   66.7%          LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      4  100.0%            LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
      2   33.3%          Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:318:19
      2  100.0%            /home/matt/.nvm/versions/node/v16.14.0/bin/node
      5   33.3%        Function: ^coverVariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:349:33
      3   60.0%          LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      3  100.0%            LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
      2   40.0%          Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:318:19
      2  100.0%            /home/matt/.nvm/versions/node/v16.14.0/bin/node
      4   26.7%        LazyCompile: *coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      4  100.0%          LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      4  100.0%            LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
      6    1.7%      LazyCompile: *expressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:318:29
      4   66.7%        Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
      4  100.0%          Function: ^coverFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:345:23
      4  100.0%            LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1   16.7%        Function: ^insertCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:177:16
      1  100.0%          Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      1  100.0%            Function: ^coverIfBranches /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:410:25
      1   16.7%        Function: ^coverSwitchCase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:435:25
      1  100.0%          LazyCompile: *wrappedEntry /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:311:45
      1  100.0%            LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
      6    1.7%      LazyCompile: *arrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:258:25
      6  100.0%        LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      6  100.0%          LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      6  100.0%            LazyCompile: *valueToNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/converters/valueToNode.js:29:21
      4    1.1%      LazyCompile: *memberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:378:26
      2   50.0%        Function: ^wrap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:173:5
      2  100.0%          LazyCompile: *increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      2  100.0%            Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      2   50.0%        Function: ^increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      1   50.0%          Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      1  100.0%            Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      1   50.0%          Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1  100.0%            Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
      4    1.1%      LazyCompile: *identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:342:20
      4  100.0%        Function: ^increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      2   50.0%          Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      2  100.0%            Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      2   50.0%          Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
      2  100.0%            Function: ^coverFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:345:23
      4    1.1%      Function: ^memberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/builders/generated/index.js:378:26
      4  100.0%        Function: ^increase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:170:11
      2   50.0%          Function: ^insertStatementCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:205:25
      1   50.0%            Function: ^coverVariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:349:33
      1   50.0%            Function: ^coverStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:333:24
      1   25.0%          Function: ^insertFunctionCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:216:24
      1  100.0%            Function: ^coverFunction /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:345:23
      1   25.0%          Function: ^getBranchIncrement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:267:21
      1  100.0%            Function: ^insertBranchCounter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:272:22
    310    1.2%    LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
    101   32.6%      LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
     99   98.0%        LazyCompile: *Property /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11122:16
     99  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     99  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      2    2.0%        Function: ^Property /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11122:16
      2  100.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      2  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     71   22.9%      LazyCompile: *Identifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6457:16
     69   97.2%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     69  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     45   65.2%            LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
     18   26.1%            LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
      2    2.9%            Function: ^BinaryExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9076:1
      1    1.4%            LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9419:16
      1    1.4%            Function: ^VariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11802:1
      1    1.4%            Function: ^NewExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11003:16
      1    1.4%            Function: ^ImportSpecifier /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10657:1
      2    2.8%        Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8157:14
      2  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      2  100.0%            Function: ^FunctionNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8017:16
     30    9.7%      LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11041:16
     30  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     30  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     27   90.0%            LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
      2    6.7%            LazyCompile: *ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8660:16
      1    3.3%            LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9419:16
     30    9.7%      LazyCompile: *Literal /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8252:1
     30  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     30  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
     24   80.0%            LazyCompile: *MethodBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:6966:16
      4   13.3%            LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
      1    3.3%            LazyCompile: *VariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11802:1
      1    3.3%            LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
     18    5.8%      LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
     18  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
     18  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      9   50.0%            LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11756:16
      6   33.3%            LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9172:16
      3   16.7%            LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9419:16
     11    3.5%      LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7905:1
     11  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      8   72.7%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      6   75.0%            LazyCompile: *BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7931:16
      1   12.5%            LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10154:14
      1   12.5%            Function: ^BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7931:16
      1    9.1%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8157:14
      1  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      1    9.1%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8143:14
      1  100.0%            Function: ^parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8157:14
      1    9.1%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10154:14
      1  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      7    2.3%      LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11756:16
      7  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      7  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      7  100.0%            LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7905:1
      5    1.6%      Function: ^BinaryExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9076:1
      5  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      5  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      3   60.0%            Function: ^SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
      1   20.0%            LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
      1   20.0%            LazyCompile: *LogicalExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10695:16
      4    1.3%      LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:9419:16
      4  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      4  100.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      1   25.0%            LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11220:1
      1   25.0%            LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7905:1
      1   25.0%            Function: ^TemplateLiteral /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:8334:1
      1   25.0%            Function: ^ReturnStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:11192:1
      4    1.3%      Function: ^IfStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10109:16
      4  100.0%        LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5137:14
      3   75.0%          LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
      3  100.0%            LazyCompile: *BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:7931:16
      1   25.0%          LazyCompile: *parseNode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:10154:14
      1  100.0%            LazyCompile: *NodeBase /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:5049:16
    298    1.2%    LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    297   99.7%      LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
    297  100.0%        LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    292   98.3%          LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
    292  100.0%            LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
      4    1.3%          LazyCompile: *crawl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:709:8
      4  100.0%            LazyCompile: *setContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:135:20
    279    1.1%    LazyCompile: *get /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/types/lib/index.js:623:19
    166   59.5%      LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     76   45.8%        LazyCompile: *ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
     70   92.1%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
     70  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      6    7.9%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      6  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     42   25.3%        LazyCompile: *printJoin /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:332:12
     39   92.9%          LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:45:26
     39  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      2    4.8%          LazyCompile: *BlockStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/base.js:31:24
      2  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1    2.4%          Function: ^printList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:400:12
      1  100.0%            Function: ^ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:45:26
     16    9.6%        LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
     16  100.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
     16  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      7    4.2%        LazyCompile: *printList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:400:12
      3   42.9%          LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:45:26
      3  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1   14.3%          LazyCompile: *CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:173:24
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1   14.3%          Function: ^VariableDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:264:29
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1   14.3%          Function: ^SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:111:28
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1   14.3%          Function: ^CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:173:24
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      4    2.4%        LazyCompile: *ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      4  100.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      4  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      3    1.8%        Function: ^ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:212:29
      2   66.7%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      2  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1   33.3%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      3    1.8%        Function: ^CallExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:173:24
      2   66.7%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      2  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1   33.3%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2    1.2%        LazyCompile: *VariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:303:28
      2  100.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      2  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2    1.2%        LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:64:26
      2  100.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      2  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2    1.2%        Function: ^ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2    1.2%        Function: ^MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2    1.2%        Function: ^ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1   50.0%          LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     59   21.1%      LazyCompile: *isFirstInContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:247:26
     55   93.2%        LazyCompile: *ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:77:26
     53   96.4%          LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     46   86.8%            LazyCompile: *ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
      4    7.5%            LazyCompile: *ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      2    3.8%            Function: ^ArrayExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:96:25
      1    1.9%            Function: ^ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
      2    3.6%          LazyCompile: *find /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/index.js:47:14
      2  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      4    6.8%        Function: ^ObjectExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:77:26
      4  100.0%          LazyCompile: *find /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/index.js:47:14
      4  100.0%            LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     13    4.7%      LazyCompile: *ObjectProperty /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/types.js:71:24
     13  100.0%        LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
     13  100.0%          LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
     13  100.0%            LazyCompile: *printJoin /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:332:12
      9    3.2%      LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:73:26
      9  100.0%        LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      7   77.8%          LazyCompile: *ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:212:29
      7  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1   11.1%          LazyCompile: *printList /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:400:12
      1  100.0%            Function: ^SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:111:28
      1   11.1%          Function: ^ExpressionStatement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:212:29
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      6    2.2%      LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
      5   83.3%        LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      5  100.0%          LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      3   60.0%            LazyCompile: *UpdateExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:64:26
      2   40.0%            LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
      1   16.7%        LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/buffer.js:186:13
      1  100.0%          LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      1  100.0%            LazyCompile: *MemberExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:257:26
      5    1.8%      LazyCompile: *checkPath /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/lib/virtual-types.js:13:12
      5  100.0%        LazyCompile: *newFn /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/visitors.js:210:26
      5  100.0%          LazyCompile: *_call /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:46:15
      5  100.0%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
      3    1.1%      LazyCompile: *VariableDeclaration /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:264:29
      3  100.0%        LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      3  100.0%          LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      3  100.0%            LazyCompile: *printJoin /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:332:12
      3    1.1%      LazyCompile: *SequenceExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/node/parentheses.js:147:28
      3  100.0%        LazyCompile: *print /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:247:8
      2   66.7%          Function: ^VariableDeclarator /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/statements.js:303:28
      2  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
      1   33.3%          LazyCompile: *AssignmentExpression /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/generators/expressions.js:227:30
      1  100.0%            LazyCompile: *withSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/generator/lib/printer.js:143:13
    265    1.0%    LazyCompile: *encode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/sourcemap-codec/dist/sourcemap-codec.umd.js:78:17
    265  100.0%      Function: ^SourceMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:181:35
    265  100.0%        Function: ^generateMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:497:58
    265  100.0%          Function: ^executeReplacement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:86:30
    265  100.0%            LazyCompile: ~renderChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:72:38
    255    1.0%    LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
    253   99.2%      LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    244   96.4%        LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
    244  100.0%          LazyCompile: *traverse.node /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:69:26
    235   96.3%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
      7    2.9%            Function: ^traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:42:18
      6    2.4%        LazyCompile: *crawl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:709:8
      6  100.0%          LazyCompile: *setContext /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/context.js:135:20
      6  100.0%            LazyCompile: *visitQueue /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/context.js:85:13
      3    1.2%        Function: ^traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/index.js:42:18
      3  100.0%          Function: ^traverse /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/path/index.js:136:11
      2   66.7%            Function: ^crawl /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@babel/traverse/lib/scope/index.js:709:8
      1   33.3%            Function: ^enter /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup-plugin-istanbul2/node_modules/istanbul-lib-instrument/dist/visitor.js:602:10

    748    2.3%  LazyCompile: *decode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:632:16
    748  100.0%    Function: ^decodedSourcemap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:14305:26
    710   94.9%      LazyCompile: ~renderChunkReducer /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:14330:32
    710  100.0%        Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22703:41
     23    3.1%      Function: ^transform /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21916:25
     23  100.0%        Function: ^addModuleSource /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22149:26
     23  100.0%          /home/matt/.nvm/versions/node/v16.14.0/bin/node
     14    1.9%      Function: ^transformReducer /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:21928:30
     14  100.0%        Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22703:41

    554    1.7%  __futex_abstimed_wait_common

    463    1.4%  LazyCompile: *encode /home/matt/Projects/darkreader_ws/_darkreader/node_modules/sourcemap-codec/dist/sourcemap-codec.umd.js:78:17
    463  100.0%    Function: ^SourceMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:181:35
    463  100.0%      Function: ^generateMap /home/matt/Projects/darkreader_ws/_darkreader/node_modules/magic-string/dist/magic-string.cjs.js:497:58
    463  100.0%        Function: ^executeReplacement /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:86:30
    463  100.0%          LazyCompile: ~renderChunk /home/matt/Projects/darkreader_ws/_darkreader/node_modules/@rollup/plugin-replace/dist/rollup-plugin-replace.cjs.js:72:38
    463  100.0%            Function: ^<anonymous> /home/matt/Projects/darkreader_ws/_darkreader/node_modules/rollup/dist/shared/rollup.js:22763:19



</details>

Reply:
> Or maybe start the echo server in a separate process?

Fine by me.

Reply:
Actually, you can accomplish most of what you need using data urls (`<link href="data:text/css..." `). You actually only need the echo server for a single test.

Reply:
That means we could start the server for just that one test

Reply:
My next commits were going to get Karma fixed up to run in GitHub Actions. Would you mind just using this band-aid until then?

Reply:
should be good enough as temporary fix for now.